app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -248,6 +269,68 @@ const Thunder = () => {
     </>
   );
 };
+// #endregion
+
+// cloud starts here
+const CloudMotionFrames = keyframes`
+   0% {
+    margin-left:-1000px;
+  }
+  100% {
+    margin-left:100%;
```

Comment:
Nice work writing the `HorizontalWeatherWrapper`! That should allow us to use `left: -200px` -> `left: 100%` instead of the `margin-left`. I think `left` absolute position is a little clearer.